hrm-service/service-tests/contacts.test.ts

```diff
@@ -435,6 +436,27 @@ describe('/contacts route', () => {
         {
           body: { firstName: 'jh', lastName: 'he' },
           changeDescription: 'multiple input search',
+          expectCallback: response => {
+            // Name based filters remove non data contacts regardless of setting?
+            expect(response.status).toBe(200);
+            const { contacts } = response.body;
+            //expect(count).toBe(2);
```

Comment:
Is this `expect` clause commented for some reason?

Reply:
I commented it out because the errors produced by the later checks are more informative when it fails

I'll move it to the bottom of the test